release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -174,6 +174,12 @@ A new `--node-selector` argument provides a way to identify which nodes you are
 
 For more information, see xref:../support/gathering-cluster-data.adoc#support-network-trace-methods_gathering-cluster-data[Network trace methods] and xref:../support/gathering-cluster-data.adoc#support-collecting-host-network-trace_gathering-cluster-data[Collecting a host network trace].
 
+[id="ocp-4-10-ne-router-compression"]
+==== Ingress Controller router compression
+This enhancement adds the ability to configure the HAProxy Ingress Controller to specify router compression globally for specific MIME types. This update enables gzip-HTTP-compression of your your ingress workloads when there are large amounts of compressible routed traffic.
```

Comment:
```suggestion
This enhancement adds the ability to configure global HTTP traffic compression on the HAProxy Ingress Controller for specific MIME types. This update enables gzip-compression of your ingress workloads when there are large amounts of compressible routed traffic.
```

Reply:
@skrthomas - something happened after this, the new text is different and incorrect.

> This enhancement adds the ability to configure global HTTP traffic compression on the Ingress Controller to specify router for specific MIME types.